bin/reth/src/cli/mod.rs

```diff
@@ -163,6 +164,9 @@ pub enum Commands<Ext: RethCliExt = ()> {
     /// Scripts for node recovery
     #[command(name = "recover")]
     Recover(recover::Command),
+    /// Script for counting opcodes occurrencies
+    #[command(name = "count-opcodes")]
+    CountOpcodes(count_opcodes::Command),
```

Comment:
I'd put this under debug or something like that since it's not really relevant to normal node ops